CHANGES.rst

```diff
@@ -24,7 +24,7 @@
 Changes
 =======
 
-Version 1.0.0b1 (released 2016-11-08)
```

Comment:
Hmm, from what Lars told me, for beta versions, we can't just bump the versions in the changelog anymore :P
We should rather create another entry with the list of changes since the previous versions (you can check how I did it here: https://github.com/inveniosoftware/dojson/commit/9ac1f416b112da04bf79634b600f07e6fb091a11)